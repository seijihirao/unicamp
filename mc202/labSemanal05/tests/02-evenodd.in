create 10
add 1
add 3
add 5
add 7
add 9
print
rank 5
rank 6
rank 7
create 10
add 2
add 4
add 6
add 8
add 10
print
rank 5
rank 6
rank 7
exit
