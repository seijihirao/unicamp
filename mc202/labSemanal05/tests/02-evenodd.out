S = {1,3,5,7,9}
rank(5) = 3
rank(6) = 3
rank(7) = 4
S = {2,4,6,8,10}
rank(5) = 2
rank(6) = 3
rank(7) = 3
